dev/com.ibm.ws.st.ui/src/com/ibm/ws/st/ui/internal/Activator.java

```diff
@@ -222,31 +223,54 @@ public void run() {
         });
 
         com.ibm.ws.st.core.internal.Activator.setFeatureConflictHandler(new FeatureConflictHandler() {
+
+            WebSphereServerInfo wsServerInfo = null;
```

Comment:
These fields are not safe.  Even if reset to null when invokeDialog is completed it is still not safe if multiple threads are calling this method at the same time.